service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java

```diff
@@ -191,7 +191,7 @@ public Integer getTagsVersion()
         {
             throw new IllegalArgumentException("tag is null");
         }
-        return this.tag.getVersionFinal();
```

Comment:
But what about the [final countdown](https://www.youtube.com/watch?v=9jK-NcRmVcw)?